tests/unit/test_work_items.py

```
@@ -253,3 +253,109 @@ def test_list_attachments(
+            # Unarchive
+            client.work_items.unarchive(workspace_slug, project.id, wi.id)
+
+            # Verify absent from archived list
+            archived_after = client.work_items.list_archived(workspace_slug, project.id)
+            archived_ids_after = [w.id for w in archived_after.results]
+            assert wi.id not in archived_ids_after
+        finally:
+            try:
+                client.work_items.delete(workspace_slug, project.id, wi.id)
+            except Exception:
+                pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)